ui/components/multichain/pages/review-permissions-page/site-cell/site-cell.tsx

```diff
@@ -24,24 +23,46 @@ import {
 } from '../../../../component-library';
 import { AvatarGroup, EditAccountsModal, EditNetworksModal } from '../../..';
 import { AvatarType } from '../../../avatar-group/avatar-group.types';
+import { useSelector } from 'react-redux';
+import { getPermittedAccountsByOrigin } from '../../../../../selectors/permissions';
 
-export const SiteCell = ({
+interface SiteCellProps {
+  networks: {
+    rpcPrefs?: { imageUrl?: string };
+    nickname: string;
+    chainId?: string;
+  }[];
+  accounts: { address: string }[];
+  onAccountsClick: () => void;
+  onNetworksClick: () => void;
+  approvedAccounts: { address: string }[];
+  activeTabOrigin: string;
+}
+
+export const SiteCell: React.FC<SiteCellProps> = ({
   networks,
   accounts,
   onAccountsClick,
   onNetworksClick,
+  approvedAccounts,
+  activeTabOrigin,
 }) => {
   const t = useI18nContext();
-  const avatarNetworksData = networks.map((network: { rpcPrefs: { imageUrl: string; }; nickname: string; }) => ({
-    avatarValue: network.rpcPrefs.imageUrl,
+  const avatarNetworksData = networks.map((network) => ({
+    avatarValue: network?.rpcPrefs?.imageUrl || '',
```

Comment:
this will need to be updated to use the network to logo constant after https://github.com/MetaMask/metamask-extension/pull/26433/ is merged